chrome/navigation.json

```diff
@@ -0,0 +1,736 @@
+{
+    "landingPage": [
+        {
+            "id": "application-services",
+            "title": "Application services",
+            "pathname": "/application-services"
```

Comment:
I'm kinda leaning towards same approach as with module defintion. Unless the `pathname` differes from `id` use the `id` field. At least for the landing page nav.

Reply:
We will use `appId` to identify to which app the link leads. `appId` will be based on the fed module ID (the insights.appname from package.json)